ccp/modules/dnpm-node-compose.yml

```diff
@@ -1,34 +1,88 @@
 version: "3.7"
 
 services:
-  dnpm-backend:
-    image: ghcr.io/kohlbacherlab/bwhc-backend:1.0-snapshot-broker-connector
-    container_name: bridgehead-dnpm-backend
+  dnpm-mysql:
+    image: mysql:latest
+    healthcheck:
+      test: [ "CMD", "mysqladmin" ,"ping", "-h", "localhost" ]
+      interval: 3s
+      timeout: 5s
+      retries: 5
     environment:
-      - ZPM_SITE=${ZPM_SITE}
-      - N_RANDOM_FILES=${DNPM_SYNTH_NUM}
+      MYSQL_ROOT_HOST: "%"
+      MYSQL_ROOT_PASSWORD: ${DNPM_MYSQL_ROOT_PASSWORD}
+    volumes:
+      - dnpm-mysql:/var/lib/mysql
+
+  dnpm-authup:
+    image: authup/authup:latest
+    container_name: bridgehead-dnpm-authup
     volumes:
-      - /etc/bridgehead/dnpm:/bwhc_config:ro
-      - ${DNPM_DATA_DIR}:/bwhc_data
+      - dnpm-authup:/usr/src/app/writable
```

Comment:
I guess we could mount it to where it was before which is this `DNPM_DATA_DIR` path that was usually in `/etc/bridgehead/dnpm` somewhere but I am not a huge fan of that location. Not sure what data is actually stored there if its patient data I would not necessarily store it under `/tmp/bridgehead` or `/var/cache/bridgehead`

Reply:
`/var/cache/` is not persistant enough for this kind of data, as it can't be reconstructed in case of a cache clearing. Obviously, `/etc/bridgehead/...` is a bad solution that was only used to use an already established path. I would propose to create a `/var/bridgehead/` directory @lablans @torbrenner ?

Reply:
Don't we have /var/lib/bridgehead for that?